src/main/java/com/capstone/producer/TfrHandler.java

```diff
@@ -0,0 +1,179 @@
+
+    private static String combineNotamMessages(String notamNumber) {
+        String[] allMessages = receivedNotams.get(notamNumber);
+        String fullMessage = "";
+        Pattern pattern = Pattern.compile("(FDC\\s*\\d/\\d{4}\\s*.*..TEMPORARY\\s*(FLIGHT)?\\s*(RESTRICTIONS.)?)(.*)(\\b\\d{10}-\\d{10})");
+        String stringToParse = "";
+        for(int i = 0; i < allMessages.length; i++) {
+            LOGGER.info("AllMessage of message: {}", allMessages[i]);
+            stringToParse = allMessages[i].replaceAll(System.getProperty("line.separator"), " ");
+            LOGGER.info("String to parse: {}", stringToParse);
+            Matcher matcher = pattern.matcher(stringToParse);
+            if(matcher.find()) {
+                fullMessage += matcher.group(4);
+            }
+        }
+        combinedNotams.put(notamNumber, fullMessage);
+        return fullMessage;
+    }
+
+    private static boolean pointRadiusParse(String notamNumber, String message) throws InterruptedException{
+        Pattern pattern = Pattern.compile("(WI\\s*AN\\s*AREA\\s*DEFINED\\s*AS\\s*(\\d*.?\\d*)NM\\s*RADIUS\\s*OF\\s*(\\d+[NS]\\d+[EW])(.*?)EFFECTIVE\\s*(\\d{10})\\s*UTC.*?UNTIL\\s*(\\d{10})?)");
+        Matcher matcher = pattern.matcher(message);
+        boolean successfulMatching = false;
+        while(matcher.find()){ //Allows for multiple finds.
+            LOGGER.info("String matched Radius test: {}", matcher.group(0));
+            JSONObject jsonObject = new JSONObject();
+            jsonObject.put("notamNumber", notamNumber);
+            jsonObject.put("type", "RADIUS");
+            jsonObject.put("latlong", matcher.group(3));
+            jsonObject.put("radius", matcher.group(2));
+            jsonObject.put("startTime", matcher.group(5));
+            if(matcher.group(6) != null) {
+                jsonObject.put("endTime", matcher.group(6));
+            } else {
+                jsonObject.put("endTime", "PERM");
+            }
+
+            KafkaProducer.runProducer(jsonObject.toString(), "TFRData");
+            successfulMatching = true;
+        }
+        return successfulMatching; //doesn't appear to be a straight circle
+    }
+
+    private static boolean boundaryParse(String notamNumber, String message) throws InterruptedException{
+        Pattern boundaryPattern = Pattern.compile("WI\\s*AN\\s*AREA\\s*DEFINED\\s*AS\\s*\\d+[NS]\\d+[EW].*?TO.*?ORIGIN.*?EFFECTIVE\\s*(\\d{10}).*?UNTIL\\s*(\\d{10})?");
+        Pattern latlongPattern = Pattern.compile("\\d+[NS]\\d+[EW]");
+        Matcher boundaryMatch = boundaryPattern.matcher(message);
+        boolean successfulMatching = false;
+        while(boundaryMatch.find()) {
+            Matcher latlongMatch = latlongPattern.matcher(boundaryMatch.group(0));
+            List<String> latlong = new ArrayList<String>();
+            while(latlongMatch.find()){
+                latlong.add(latlongMatch.group(0));
+            }
+
+            JSONObject jsonObject = new JSONObject();
```

Comment:
Can you replace json object with @data classes that you serialize to json using the jasckson object mapper. It makes it easier to know at a glance the structure of the object that we send as json

Reply:
Reasonably sure my latest push addressed this comment. Please lmk if you want something else done with that.